src/tools/run-make-support/src/lib.rs

```diff
@@ -296,7 +296,8 @@ pub fn not_contains<P: AsRef<Path>>(path: P, expected: &str) -> bool {
 }
 
 /// Returns true if the filename at `path` is not in `expected`.
-pub fn filename_not_in_denylist<P: AsRef<Path>>(path: P, expected: &[String]) -> bool {
+pub fn filename_not_in_denylist<P: AsRef<Path>, V: AsRef<[String]>>(path: P, expected: V) -> bool {
```

Comment:
I'm not sure if we need to stuff these generics everywhere, as it makes the compilation of each run-make test slightly slower. But if the function was already generic before, it doesn't really change anything, so should be fine here.